django/db/models/sql/query.py

```diff
@@ -1058,7 +1058,7 @@ def get_external_cols(self):
 
     def as_sql(self, compiler, connection):
         if self.subquery and not connection.features.supports_ordered_subquery:
-            self.clear_ordering(force=True)
+            self.clear_ordering()
```

Comment:
Had to remove the `force=True` here since it caused failures on Oracle where actual queries was executed 🤔 . Weird stuff! Anyway not using `force` here brings Oracle back to the previous behaviour. If anyone is better versed in Oracle than me, these where the failing tests:

* `annotations.tests.NonAggregateAnnotationTestCase.test_annotation_subquery_outerref_transform`
* `expressions_window.tests.WindowFunctionTests.test_subquery_row_range_rank`
* `queries.tests.QuerySetBitwiseOperationTests.test_or_with_both_slice_and_ordering`
* `queries.tests.SubqueryTests.test_distinct_ordered_sliced_subquery`
* `queries.tests.SubqueryTests.test_ordered_subselect`
* `queries.tests.SubqueryTests.test_related_sliced_subquery`
* `queries.tests.SubqueryTests.test_slice_subquery_and_query`
* `queries.tests.SubqueryTests.test_sliced_delete`
* `select_for_update.tests.SelectForUpdateTests.test_ordered_select_for_update`

Reply:
I think it should be good enough for now if it brings Oracle back to what it was, I was also confused by these failures.

Reply:
> Weird stuff!

Oracle doesn't ignore unnecessary `ORDER BY` clauses in subqueries, i.e. it raises an error if a subquery doesn't use `DISTINCT`, `FOR UPDATE`, or is not sliced. I renamed `supports_ordered_subquery` to `ignores_unnecessary_order_by_in_subquery`, it's more accurate, IMO.

Reply:
> Oracle doesn't ignore unnecessary ORDER BY clauses in subqueries, i.e. it raises an error if a subquery doesn't use DISTINCT, FOR UPDATE, or is not sliced.

Ah yes that explains it, must've missed the slices when I looked at the failed tests.

> I renamed supports_ordered_subquery to ignores_unnecessary_order_by_in_subqueries, it's more accurate, IMO

Yep! 👍